templates/pages/open.html

```diff
@@ -89,7 +115,7 @@ <h2>{% trans "Users who have contributed over ten packages" %}</h2>
                 </table>
             </div>
             <div class="col-sm-6 col-md-5">
-                <h2>{% trans "Grids with more than fifteen packages" %}</h2>
+                <h2>{% trans "Grids with twenty or more packages" %}</h2>
```

Comment:
I noticed that the code was updated to ‘ num_packages__gte=25’ but the copy displayed on the page says twenty or more. Not sure if that **really** matters or not

Reply:
Nice catch. Thank you. I was visually making changes and fiddling, and that one got away from me.